client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -219,6 +219,37 @@ export const StockLineForm = ({
               />
             }
           />
+          <StyledInputRow
```

Comment:
Moved this so the two sections are more even... this seemed to make the most sense to be near batch and expiry